src/ui/libs/DatalensChartkit/ChartKit/plugins/Table/renderer/TableWidget.tsx

```diff
@@ -131,20 +133,22 @@ const TableWidget = React.forwardRef<ChartKitWidgetRef | undefined, TableWidgetP
                 data-qa={ChartKitTableQa.Widget}
                 ref={ref}
             >
-                {titleText && <div className={b('title')}>{titleText}</div>}
-                <div className={b('table-wrapper', {'highlight-rows': shouldHighlightRows})}>
-                    {dimensions?.width && (
-                        <Table
-                            data={tableData}
-                            noData={{text: i18n('chartkit-table', 'message-no-data')}}
-                            onCellClick={onCellClick}
-                            header={{
-                                sticky: true,
-                            }}
-                            manualSorting={isPaginationEnabled}
-                            onSortingChange={isPaginationEnabled ? onSortingChange : undefined}
-                        />
-                    )}
+                <div className={[b('snapter-container'), SNAPTER_HTML_CLASSNAME].join(' ')}>
```

Comment:
May be added as mixin in b()
<img width="492" alt="2024-05-21_11-34-58" src="https://github.com/datalens-tech/datalens-ui/assets/56549651/13c3b64c-bcee-46c1-942b-96bd50148ee4">